src/utility.jl

```diff
@@ -44,7 +44,7 @@ end
 
 
 """
-Estimate the upper bound with the Monte-Carlo error
+Estimate the upper bound with a distribution of costs
 
```

Comment:
Give detail on the choosen error : 
the expectation of the costs have 97.5% chance of being under the upper bound.  

I would like to see the 1.96 not being so hard-coded, at some point we might want to
allow the user to change this parameter in one place (a global parameter in the package ?).

For further reference if you want P(E(cost) < upb )>= p, you need to use
sqrt(2) * erfinv(2 * p - 1.0) to compute the quantile.